Update packages.markdown

ArveVM opened the pull request:
## Proposed change
Update information and examples of Packages-folder

Better information to users on two different methods, and added more samples to visualize the differences.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

ArveVM committed:
Update packages.markdown

ArveVM committed:
Merge branch 'current' into patch-2

frenck reviewed (changes requested):
I consider myself pretty well familiar with packages, all YAML includes and possible structures one can make; however, I'm completely lost on what you are trying to show or say in this PR.

Why does this matter or need to be explained this way? And two options? What does that even mean?

ArveVM commented:
> I consider myself pretty well familiar with packages, all YAML includes and possible structures one can make; however, I'm completely lost on what you are trying to show or say in this PR.
> 
> Why does this matter or need to be explained this way? And two options? What does that even mean?

Sorry for the confusion. 
Trying to clarify the current presented options; !include_dir_named and !include_dir_merge_named (so two options are already presented - but for noobs, it is not easy to see what is what, and examples presented currently for only one option).

As it is stated in the docs now,,, it seems like include_dir_named will include with keys representing filename.

When I've asked friends to use packages (and to read the docs!) -they are not selecting include_dir_named because it supposedly need keys representing filename.
I think it is the prefered solution, hence putting it first and adding the 'as if the content was written in configuration.yaml' which I see as a huge benefit.
 
Another/better solution could be to skip all references to include_dir_merge_named and just keep the link to splitting the configuration?? - Then experts will sort them selves out?

would you like me to have another shot at clarifying the two options better (or not call them options?) - or to remove include_dir_merge_named 'option' ?

ArveVM committed:
simplification

Removed the two options,, focused on only the simplest (best) as from Frenck's example
Added info on indentation as if in configuration.yaml,, to explain why this should be primary option..

ArveVM committed:
Merge branch 'current' into patch-2

ArveVM commented:
> I consider myself pretty well familiar with packages, all YAML includes and possible structures one can make; however, I'm completely lost on what you are trying to show or say in this PR.
> 
> Why does this matter or need to be explained this way? And two options? What does that even mean?

have tried to simplify; 
- removed the option for !include_dir_merge_named from origninal doc, so now this is all about !include_dir_named
- adjusted package sample to the syntax/indentation of !include_dir_named 
- added info on this way of including keep indentation "as if written in configuration.yaml"


And the purpose of this PR is to be able to refer to documentation when urging people to use "packages-folder" with the indentation of configuration.yaml - just as you do  :)
Currently I've seen several examples where they have read the documentation and set up with !include_dir_merge_named and having to keep up with different indentation than most examples that can be found in communities etc.

ArveVM committed:
Merge branch 'current' into patch-2

ArveVM committed:
Merge branch 'current' into patch-2
